### PR TITLE
Fix bug in help message & Fix bug when using ip command in old version & Update doc.

### PR DESCRIPTION
**Pull request by Andrewmatilde**
The help message in old version will print `Error` ahead of logging help message .
I have tried to run rs-tproxy on centos 8+ & centos 7.9 & ubuntu 20LTS & ubuntu18LTS & Debian 10  , and then I find some problem with ebtables version & 'ip' command version.
I fixed these problem in this pr.

Update doc:
- [x] Add detailed info of config file.

### CODE REVIEW

**Comment by Andrewmatilde** [comment]
PTAL, @Hexilee , @cwen0 .

**Review by Hexilee — APPROVED** [review]
LGTM!

**Comment by Andrewmatilde** [comment]
> Please execute `make fmt` before merging.

Fixed.

File: README.md [hunk 1]
```diff
@@ -1,30 +1,117 @@
 # rs-tproxy
 
-Transparent proxy in only linux 2.0+ platform for injecting [Abort|Delay|Append|Replace] HTTP packet.
+Transparent HTTP proxy for [Abort|Delay|Append|Replace] packet.
```
Comment (cwen0) [inline README.md:3]: We need a more detailed doc to help new users using rs-tproxy.

Comment (Andrewmatilde) [reply README.md:3]: I think so. Should we add doc in website of chaos-mesh?

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**